src/plugins/intel_gpu/src/plugin/ops/scaled_dot_product_attention.cpp

```diff
@@ -0,0 +1,59 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "intel_gpu/plugin/program_builder.hpp"
+#include "intel_gpu/plugin/common_utils.hpp"
+
+#include "intel_gpu/op/sdpa.hpp"
+
+#include "openvino/op/scaled_dot_product_attention.hpp"
+
+#include "intel_gpu/primitives/scaled_dot_product_attention.hpp"
+
+namespace ov {
+namespace op {
+namespace internal {
+using SDPA = ov::intel_gpu::op::SDPA;
+}  // namespace internal
+}  // namespace op
+}  // namespace ov
+
+namespace ov {
+namespace intel_gpu {
+
+static void CreateScaledDotProductAttentionOp(ProgramBuilder& p, const std::shared_ptr<ov::op::v13::ScaledDotProductAttention>& op) {
```

Comment:
I think we can always convert to internal SDPA to avoid handling of both ops here